lib/sisimai/bite/email/messagelabs.rb

```diff
@@ -0,0 +1,213 @@
+              # Please visit www.messagelabs.com/support for more details
+              # about this error message and instructions to resolve this issue.
+              #
+              #
+              # --b0Nvs+XKfKLLRaP/Qo8jZhQPoiqeWi3KWPXMgw==
+              # Content-Type: message/delivery-status
+              #
+              # Reporting-MTA: dns; server-15.bemta-3.messagelabs.com
+              # Arrival-Date: Tue, 23 Dec 2014 20:39:34 +0000
+              #
+              # Action: failed
+              # Status: 5.0.0
+              # Last-Attempt-Date: Tue, 23 Dec 2014 20:39:35 +0000
+              # Remote-MTA: dns; mail.dest.example.net
+              # Diagnostic-Code: smtp; 550 maria@dest.example.net... No such user
+              # Final-Recipient: rfc822; maria@dest.example.net
+              v = dscontents[-1]
+
+              if cv = e.match(/\A[Ff]inal-[Rr]ecipient:[ ]*(?:RFC|rfc)822;[ ]*([^ ]+)\z/)
+                # Final-Recipient: rfc822; maria@dest.example.net
+                if v['recipient']
+                  # There are multiple recipient addresses in the message body.
+                  dscontents << Sisimai::Bite.DELIVERYSTATUS
+                  v = dscontents[-1]
+                end
+                v['recipient'] = cv[1]
+                recipients += 1
+
+              elsif cv = e.match(/\A[Aa]ction:[ ]*(.+)\z/)
+                # Action: failed
+                v['action'] = cv[1].downcase
+
+              elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+                # Status: 5.0.0
+                v['status'] = cv[1]
+
+              else
+                if cv = e.match(/\A[Dd]iagnostic-[Cc]ode:[ ]*(.+?);[ ]*(.+)\z/)
+                  # Diagnostic-Code: smtp; 550 maria@dest.example.net... No such user
+                  v['spec'] = cv[1].upcase
+                  v['diagnosis'] = cv[2]
+
+                elsif p =~ /\A[Dd]iagnostic-[Cc]ode:[ ]*/ && cv = e.match(/\A[ \t]+(.+)\z/)
+                  # Continued line of the value of Diagnostic-Code header
+                  v['diagnosis'] ||= ''
+                  v['diagnosis']  += ' ' + cv[1]
+                  havepassed[-1] = 'Diagnostic-Code: ' + e
+                end
+              end
+
+            else
+              # Reporting-MTA: dns; server-15.bemta-3.messagelabs.com
+              # Arrival-Date: Tue, 23 Dec 2014 20:39:34 +0000
+              if cv = e.match(/\A[Rr]eporting-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                # Reporting-MTA: dns; server-15.bemta-3.messagelabs.com
+                next if connheader['lhost'].size > 0
+                connheader['lhost'] = cv[1].downcase
+                connvalues += 1
+
+              elsif cv = e.match(/\A[Aa]rrival-[Dd]ate:[ ]*(.+)\z/)
+                # Arrival-Date: Tue, 23 Dec 2014 20:39:34 +0000
+                next if connheader['date'].size > 0
+                connheader['date'] = cv[1]
+                connvalues += 1
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->